arkouda/series.py

```diff
@@ -525,20 +540,26 @@ def concat(arrays, axis=0, index_labels=None, value_labels=None):
         if len(arrays) == 0:
             raise IndexError("Array length must be non-zero")
 
-        if type(next(iter(arrays))) == tuple:
+        types = set([type(x) for x in arrays])
+        if len(types) != 1:
+            raise TypeError(f"Items must all have same type: {types}")
+
+        if isinstance(arrays[0], tuple):
             arrays = [Series(i) for i in arrays]
 
         if axis == 1:
             # Horizontal concat
             if value_labels is None:
-                value_labels = [f"val_{i}" for i in range(len(arrays))]
+                value_labels = ["val_{}".format(i) for i in range(len(arrays))]
```

Comment:
Can we update this to use `f` string as well so
```python
value_labels = [f"val_{i}" for i in range(len(arrays))]
```

Reply:
Done